test/integration/integration_test.go

```diff
@@ -394,6 +429,22 @@ func variableShouldNotBeEmpty(variableName string) error {
 	return compareExpectedWithActualNotEquals("", minishift.GetVariableByName(variableName).Value)
 }
 
+func commandReturnShouldMatch(commandField string, expected string) error {
+	return compareExpectedWithActualMatches(expected, selectFieldFromLastOutput(commandField))
+}
+
+func commandReturnShouldNotMatch(commandField string, notexpected string) error {
+	return compareExpectedWithActualNotMatches(notexpected, selectFieldFromLastOutput(commandField))
+}
+
+func commandReturnShouldMatchContent(commandField string, expected *gherkin.DocString) error {
+	return compareExpectedWithActualMatches(expected.Content, selectFieldFromLastOutput(commandField))
```

Comment:
@hferentschik Yes, tried it and fortunately it seems that it works in golang just fine, have not find a problem yet, tested with:

```gherkin
  Scenario: Test
    When executing "minishift addons list"
    Then stdout should match "^- .* : .* P\(0\)\n- .* : .* P\(0\)\n- .* : .* P\(0\)\n$"
    Then stdout should match
    """
    ^- .* : .* P\(0\)
    - .* : .* P\(0\)
    - .* : .* P\(0\)\n$
    """
``` 

Test passes.